src/pages/docs/approvals/jira-service-management/index.md

```diff
@@ -128,9 +139,28 @@ To enable a project to enforce a requirement for an approved CR:
 ![JSM Integration Project settings](/docs/approvals/jira-service-management/images/jsm-project-settings.png)
 :::
 
+### Setting up runbooks for CR approval
+
+:::div{.warning}
+This feature is only available for version **2025.2.x** and later
+:::
+
+To enforce a runbook run to require and approved CR, the **Change Controlled** settings needs to be enabled in **both** the project and the environment the runbooks is run in and additionally the runbook needs to be included in the **Enabled Runbooks** setting.
```

Comment:
```suggestion
To enforce a runbook run to require an approved CR, the **Change Controlled** setting needs to be enabled in **both** the project and the environment the runbook is run in and additionally the runbook needs to be included in the **Enabled Runbooks** setting.
```

Reply:
You could reorder this if you wanted, it weirdly makes it read a bit better for me:

```suggestion
To enforce a runbook run to require an approved CR, the runbook needs to be included in the **Enabled Runbooks** setting and the **Change Controlled** setting needs to be enabled in **both** the project and the environment the runbook is run in. 
```